README.md

```diff
@@ -1,37 +1,103 @@
-# MarkLogic Unit Test
+![GitHub release](https://img.shields.io/github/release/marklogic-community/marklogic-unit-test.svg)
+![GitHub last commit](https://img.shields.io/github/last-commit/marklogic-community/marklogic-unit-test.svg)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+# Testing MarkLogic
 
-marklogic-unit-test includes the [original Roxy unit test framework for MarkLogic](https://github.com/marklogic-community/roxy/wiki/Unit-Testing) and 
-provides a few new features:
+marklogic-unit-test is an [ml-gradle bundle](https://github.com/marklogic-community/ml-gradle/wiki/Bundles) that allows
```

Comment:
I don't think we should mention the bundle concept yet, it can confuse the user and also make them think they can't use marklogic-unit-test without ml-gradle. The bundle concept should be brought up when showing the user how to make use of marklogic-unit-test - e.g. "Hey look how easy it is to add to your project, whether you're using DHF or a regular ml-gradle project". 

I'd start this as "marklogic-unit-test makes it easy to write tests both against endpoints you've developed within MarkLogic and against the library functions that power those endpoints. Tests written using marklogic-unit-test can easily be run using a custom endpoint in MarkLogic, or they can be run as part of a larger suite of tests using a test framework like JUnit." And then dive into how to get started.

Reply:
I removed references to ml-bundles.